.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+exclude: ^docs/
```

Comment:
If I am not wrong, this prevents running pre-commit on any file inside docs dir?

Reply:
yeah, we can see if including docs in linters would be beneficial or not, this was done as an experiment we can check if mdx is also parsed by pre-comit.

Reply:
okay, let me have a look.